maintainers/maintainer-list.nix

```diff
@@ -661,6 +661,12 @@
     githubId = 3965744;
     name = "Arthur Lee";
   };
+  arturcygan = {
+    email = "arczicygan@gmail.com";
+    github = "arcz";
+    githubId = 4679721;
+    name = "Artur Cygan";
+  };
```

Comment:
Please make this into a separate commit:

````
maintainers: add arturcygan
```

Reply:
Done